lib/src/main/java/uk/gov/di/ipv/core/library/validation/AuthRequestValidator.java

```diff
@@ -0,0 +1,86 @@
+package uk.gov.di.ipv.core.library.validation;
```

Comment:
I like this a lot. Much better way to validate the request. Maybe something we could use on other Handlers.